keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -336,7 +444,7 @@ def load(
             download the data file, and unzip. It will be used as a cach directory.
             Default to None, and `~/.keras/pascal_voc_2012` will be used.
     """
-    supported_split_value = ["train", "eval", None]
+    supported_split_value = ["train", "eval", "trainval", "sbd_train", "sbd_eval"]
```

Comment:
One other API approach to consider:
load(`split="train", semantic_boundaries=True`)

(As a user, I feel like whether or not I'm using semantic boundaries is a little bit orthogonal to which split I'm using) What do you think?

Reply:
(Regardless of exact API decision, I think some docs on the `load` method about the meaning of `sbd` would be good)

Reply:
The original tfds proposal is to use "sbd_train" to differentiate with voc data, see [here](https://github.com/AdamHillier/datasets/blob/38a1b5d73c4e6e3700d77fbe31c9cc469f3f5918/tensorflow_datasets/object_detection/voc.py#L80)

Either way seems fine to me -- we could probably even just get rid of voc data for segmentation tasks.